docs/index.asciidoc

```diff
@@ -75,5 +78,36 @@ The IP network(s) to check against. Example:
     }
 
 
+[id="plugins-{type}s-{plugin}-network_path"]
+===== `network_path`
+
+  * Value type is <<path,path>>
```

Comment:
It's not?

`:validate => :path` is a valid config option for plugins

And for this kind of text in the docs, I see 62 instances:

```
% grep 'Value type is <<path,path>>' logstash-*/docs/index.asciidoc  | wc -l
62

% grep 'Value type is <<path,path>>' logstash-*/docs/index.asciidoc
logstash-codec-netflow/docs/index.asciidoc:  * Value type is <<path,path>>
logstash-codec-netflow/docs/index.asciidoc:  * Value type is <<path,path>>
logstash-codec-netflow/docs/index.asciidoc:  * Value type is <<path,path>>
logstash-filter-elasticsearch/docs/index.asciidoc:  * Value type is <<path,path>>
logstash-filter-geoip/docs/index.asciidoc:  * Value type is <<path,path>>
...
```
```